### PR TITLE
global: display sizes in human readable across UI

### PR DESCRIPTION
**Pull request by diegodelemos**
* Closes reanahub/reana-server#296.

### CODE REVIEW

File: reana-ui/src/components/WorkflowActionsPopup.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default function WorkflowActionsPopup({ workflow, className }) {
   const [open, setOpen] = useState(false);
   const { id, size, status, session_status: sessionStatus } = workflow;
   const isDeleted = status === "deleted";
-  const isDeletedUsingWorkspace = isDeleted && size !== "0K";
+  const isDeletedUsingWorkspace = isDeleted && size.raw !== 0;
```
Comment (mvidalgarcia) [inline reana-ui/src/components/WorkflowActionsPopup.js:36]: I guess something like this is enough:

```suggestion
  const isDeletedUsingWorkspace = isDeleted && size.raw;
```

File: reana-ui/src/pages/workflowList/components/WorkflowList.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default function WorkflowList({ workflows, loading }) {
           session_status: sessionStatus,
         } = workflow;
         const isDeleted = status === "deleted";
-        const isDeletedUsingWorkspace = isDeleted && size !== "0K";
+        const isDeletedUsingWorkspace = isDeleted && size.raw !== 0;
```
Comment (mvidalgarcia) [inline reana-ui/src/pages/workflowList/components/WorkflowList.js:54]: Same

**Review by mvidalgarcia — APPROVED** [review]

File: reana-ui/src/pages/workflowList/components/WorkflowList.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ export default function WorkflowList({ workflows, loading }) {
                 }`}
               >
                 <Icon name="hdd" />
-                {size}
+                {size.human_readable}
```
Comment (mvidalgarcia) [inline reana-ui/src/pages/workflowList/components/WorkflowList.js:77]: I see that while the workflow is running we still don't have its size but the icon shows anyway.

image

WDYT about hiding the icon if we don't have the human-readable value?

Comment (mvidalgarcia) [reply reana-ui/src/pages/workflowList/components/WorkflowList.js:77]: Also, nor necessarily belonging to this PR, but we should display the size on the details page as well, we're not doing it at the moment. This comment can be converted into an issue if you find this appropriate.

Comment (diegodelemos) [reply reana-ui/src/pages/workflowList/components/WorkflowList.js]: > I see that while the workflow is running we still don't have its size but the icon shows anyway.

Fixed, thanks!

> Also, not necessarily belonging to this PR, but we should display the size on the details page as well, we're not doing it at the moment. This comment can be converted into an issue if you find this appropriate.

Good catch! Let's do this in another issue, just to not delay this PR group more.

**Pull request merged**